src/capitalsFirst.js

```diff
@@ -19,6 +19,20 @@
  */
 function capitalsFirst(str) {
   // write code here
+  let strOfUpperCase = [];
+  let strOfLowerCase = [];
+
+  str.split(' ').forEach(someWord => {
+    if (someWord[0] >= 'A' && someWord[0] <= 'Z') {
+      strOfUpperCase.push(someWord);
+      return 1;
```

Comment:
You are mixed 2 solutions. Return from forEach doing nothing, maybe you wanted to sort the array, if yes think about filter it before sort

Reply:
Yes, it was sort... ) I really don't know how I missed sort -> forEach and don't know when I did that).
Fixed.